src/perfectionist.js

```diff
@@ -1,23 +1,37 @@
 const customGrouping = {
   'custom-groups': {
     id: '{id,uuid}',
+    title: 'title',
+    name: 'name',
+    path: 'path',
     booleans: '{is*,has*}',
     date: '{*At,*On,*_at,*_on,*_date,*_time,*Date,*Time}',
     event: 'on*',
     relations: '{*Id,*_id,*_uuid,*_uuid}',
+    component: 'component',
+    meta: 'meta',
+    redirect: 'redirect',
+    children: 'children',
+    unknown: '*',
     routeFirst: '{path,route}',
```

Comment:
Als ge die zo doet zou ik die routeFirst / routeLast eruit halen, dit is anders heel verwarrend om meerdere namen op verschillende volgordes te zetten

Reply:
en ik zou unknown sowieso boven component / meta / redirect / children zetten, dit gaat alles orderen en volgens mij mogen andere waardes boven die staan